Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -265,41 +265,36 @@ public struct TestRequirement: Decodable {
     }
 }
 
-extension BSONDocument {
+public protocol SortedEquatable {
+    func sortedEquals(_ other: Self) -> Bool
+}
+
+extension BSONDocument: SortedEquatable {
     public func sortedEquals(_ other: BSONDocument) -> Bool {
```

Comment:
We have an implementation of this in `swift-bson` now (called `equalsIgnoreKeyOrder`), but it currently has a bug where it doesn't compare documents contained in an array via `equalsIgnoreKeyOrder`. I filed SWIFT-1086 to cover fixing that. Once that's done we can remove this implementation of `sortedEquals` and just call through to `equalsIgnoreKeyOrder`.